gui/src/boardbno055.cpp

```diff
@@ -188,7 +189,7 @@ void BoardBNO055::parseIncomingHT(QString cmd)
     // Setting Data
     else if(cmd.left(5) == "$SET$") {
         QStringList setd = cmd.right(cmd.length()-5).split(',',Qt::KeepEmptyParts);
-        if(setd.length() == 22) {
+        if(setd.length() == 23) {
```

Comment:
i don't quite understand how this field count works here when i add a new field.  please review and let me know

Reply:
No need to change this. This file is actually for the Original Arduino Nano + BNO055 IMU nothing to to with the BLE33. You can remove all these changes in this file. (Pretty much old code now, although it still does connect and work with it. FYI https://github.com/dlktdr/HeadTracker/wiki/BNO055-Build-Instructions-(Discontinued)

BLE33 is all sent with the JSON settings and is dynamic. Should need to change anything here either but boardnanoble33.cpp is how it communicates with the board if you want to see.

Reply:
ok, i'll revert the changes to this file.